environment/config.go

```diff
@@ -51,6 +55,41 @@ type Configuration struct {
 
 	// StaticConfiguration is used for running M3DB with a static config
 	Static *StaticConfiguration `yaml:"static"`
+
+	// Presence of a (etcd) server in this config denotes an embedded cluster
+	SeedNode *SeedNodeConfig `yaml:"seedNode"`
+
+	// NamespaceResolutionTimeout is the maximum time to wait to discover namespaces from KV
+	NamespaceResolutionTimeout time.Duration `yaml:"namespaceResolutionTimeout"`
+}
+
+// SeedNodeConfig defines fields for seed node
+type SeedNodeConfig struct {
+	RootDir                  string                 `yaml:"rootDir"`
+	InitialAdvertisePeerUrls []string               `yaml:"initialAdvertisePeerUrls"`
+	AdvertiseClientUrls      []string               `yaml:"advertiseClientUrls"`
+	ListenPeerUrls           []string               `yaml:"listenPeerUrls"`
+	ListenClientUrls         []string               `yaml:"listenClientUrls"`
+	InitialCluster           []SeedNode             `yaml:"initialCluster"`
+	Name                     string                 `yaml:"name"`
+	ClientTransportSecurity  SeedNodeSecurityConfig `json:"clientTransportSecurity"`
+	PeerTransportSecurity    SeedNodeSecurityConfig `json:"peerTransportSecurity"`
```

Comment:
Is there a reason we're using `json` field tags here?  Do we ever parse from JSON?  If not can we change these to `yaml` tags?

Reply:
Nice catch, thanks. Typo'd here.